app/scripts/controllers/membership.js

```diff
@@ -26,8 +26,11 @@ angular
       var allRoles = [];
 
       // NOTE: these could all be moved out into a strings service.
-      var messages = {
-        errorReason: _.template('Reason: "<%- httpErr %>"'),
+      var unescaped = {
+        errorReason: _.template('Reason: "<%= httpErr %>"')
+      };
+
+      var escaped = {
```

Comment:
Well, since the real issue here seems to be security, the word 'escaped' seems to be more descriptive if the specific problem we are trying to solve.  If I came back in a month I may wonder "why do I care which ones have markup?" and would have to dig. The word  'escaped' immediately tells me this is important for security.